pkg/apis/serving/v1beta1/inference_service_validation.go

```diff
@@ -109,10 +110,6 @@ func validateInferenceService(isvc *InferenceService) (admission.Warnings, error
 		return allWarnings, err
 	}
 
-	if err := validateAutoscalerTargetUtilizationPercentage(isvc); err != nil {
```

Comment:
deprecate the metrics target annotation as we already have explicit field and autoscaling spec